Babylon/groups/api/groups/connector/commands/delete.py

```diff
@@ -0,0 +1,52 @@
+from logging import getLogger
+
+from click import argument, command, confirm, make_pass_decorator, prompt
+from cosmotech_api.api.connector_api import ConnectorApi
+from cosmotech_api.exceptions import NotFoundException, UnauthorizedException
+
+from ......utils.decorators import allow_dry_run, timing_decorator
+
+logger = getLogger("Babylon")
+
+pass_connector_api = make_pass_decorator(ConnectorApi)
+
+
+@command()
+@allow_dry_run
+@timing_decorator
+@pass_connector_api
+@argument("connector_id", type=str)
+def delete(
+    connector_api: ConnectorApi,
+    connector_id: str,
+    dry_run: bool = False,
+):
+    """Unregister a connector via Cosmotech API."""
+    try:
+        if not dry_run:
+            _ = connector_api.find_connector_by_id(connector_id)  #
+        if confirm(
+            f"You are trying to delete connector {connector_id} \nDo you want to continue ?"
+        ):
+            confirm_connector_id = prompt("Confirm connector id ")
+
+            if confirm_connector_id == connector_id:
+                if dry_run:
+
+                    logger.info(
+                        "DRY RUN - Would call connector_api.unregister_connector"
+                    )
```

Comment:
You certainly can just return at this point, so you don't need the else.
I think if you install the pylint extension it will help avoiding those.